scripts/postlink/ios/postlink.js

```diff
@@ -4,8 +4,20 @@ var inquirer = require('inquirer');
 var path = require("path");
 var plist = require("plist");
 
+// Fix for https://github.com/Microsoft/react-native-code-push/issues/477
+// Try to resolve package.json, assuming we are working off of the project root
+// We will use this later to fetch the correct AppDelegate and Plist file, in case there are multiple.
+var package = require(path.resolve('./package.json'));
```

Comment:
Right on the mark @silhouettes: it just requires off of `__dirname`. I must have forgotten a `../` somewhere (a mistake easily made when the file is that deep).

Let's keep things idiomatic and just use a normal `require`. PR update coming in.